In [3]:
import pandas as pd

In [4]:
df = pd.read_csv('SYB63_314_202009_Internet_Usage.csv', skiprows=1)

In [5]:
df.head(142)

Region/Country/Area                     Unnamed: 1  Year  \
0                      1  Total, all countries or areas  2000   
1                      1  Total, all countries or areas  2005   
2                      1  Total, all countries or areas  2010   
3                      1  Total, all countries or areas  2015   
4                      1  Total, all countries or areas  2016   
..                   ...                            ...   ...   
137                    9                        Oceania  2017   
138                   53      Australia and New Zealand  2000   
139                   53      Australia and New Zealand  2005   
140                   53      Australia and New Zealand  2010   
141                   53      Australia and New Zealand  2015   

                                           Series    Value Footnotes  \
0    Percentage of individuals using the internet   6.5316       NaN   
1    Percentage of individuals using the internet  15.6777       NaN   
2    Percentage of individuals using the internet  28.7084       NaN   
3    Percentage of individuals using the internet  41.5491       NaN   
4    Percentage of individuals using the internet  44.5569       NaN   
..                                            ...      ...       ...   
137  Percentage of individuals using the internet  68.1514       NaN   
138  Percentage of individuals using the internet  46.8614       NaN   
139  Percentage of individuals using the internet  62.9524       NaN   
140  Percentage of individuals using the internet  76.7348       NaN   
141  Percentage of individuals using the internet  85.1525       NaN   

                                                Source  
0    International Telecommunication Union (ITU), G...  
1    International Telecommunication Union (ITU), G...  
2    International Telecommunication Union (ITU), G...  
3    International Telecommunication Union (ITU), G...  
4    International Telecommunication Union (ITU), G...  
..                                                 ...  
137  International Telecommunication Union (ITU), G...  
138  International Telecommunication Union (ITU), G...  
139  International Telecommunication Union (ITU), G...  
140  International Telecommunication Union (ITU), G...  
141  International Telecommunication Union (ITU), G...  

[142 rows x 7 columns]

In [6]:
df.columns 

Index(['Region/Country/Area', 'Unnamed: 1', 'Year', 'Series', 'Value',
       'Footnotes', 'Source'],
      dtype='object')

In [7]:
df [['Region/Country/Area', 'Unnamed: 1', 'Year', 'Series', 'Value']]

Region/Country/Area                     Unnamed: 1  Year  \
0                       1  Total, all countries or areas  2000   
1                       1  Total, all countries or areas  2005   
2                       1  Total, all countries or areas  2010   
3                       1  Total, all countries or areas  2015   
4                       1  Total, all countries or areas  2016   
...                   ...                            ...   ...   
1464                  716                       Zimbabwe  2005   
1465                  716                       Zimbabwe  2010   
1466                  716                       Zimbabwe  2015   
1467                  716                       Zimbabwe  2016   
1468                  716                       Zimbabwe  2017   

                                            Series    Value  
0     Percentage of individuals using the internet   6.5316  
1     Percentage of individuals using the internet  15.6777  
2     Percentage of individuals using the internet  28.7084  
3     Percentage of individuals using the internet  41.5491  
4     Percentage of individuals using the internet  44.5569  
...                                            ...      ...  
1464  Percentage of individuals using the internet   2.4000  
1465  Percentage of individuals using the internet   6.4000  
1466  Percentage of individuals using the internet  22.7428  
1467  Percentage of individuals using the internet  23.1200  
1468  Percentage of individuals using the internet  27.0555  

[1469 rows x 5 columns]

les 142 lignes portent sur des zones consolidées 
A partir de la 138, par pays // Australia

In [8]:
df2 = pd.read_csv('SYB63_315_202009_Water_and_Sanitation_Services.csv', skiprows=1)

In [9]:
df2.head(262)

Region/Country/Area                     Unnamed: 1  Year  \
0                      1  Total, all countries or areas  2005   
1                      1  Total, all countries or areas  2005   
2                      1  Total, all countries or areas  2005   
3                      1  Total, all countries or areas  2005   
4                      1  Total, all countries or areas  2005   
..                   ...                            ...   ...   
257                    9                        Oceania  2017   
258                   53      Australia and New Zealand  2005   
259                   53      Australia and New Zealand  2005   
260                   53      Australia and New Zealand  2010   
261                   53      Australia and New Zealand  2010   

                                                Series    Value Footnotes  \
0    Safely managed drinking water sources, urban (...  86.1804       NaN   
1    Safely managed drinking water sources, rural (...  43.0323       NaN   
2    Safely managed drinking water sources, total (...  64.2425       NaN   
3    Safely managed sanitation facilities, urban (P...  37.0176       NaN   
4    Safely managed sanitation facilities, rural (P...  26.0720       NaN   
..                                                 ...      ...       ...   
257  Safely managed sanitation facilities, total (P...  52.9438       NaN   
258  Safely managed drinking water sources, urban (...  93.8530       NaN   
259  Safely managed sanitation facilities, total (P...  61.9383       NaN   
260  Safely managed drinking water sources, urban (...  95.7497       NaN   
261  Safely managed sanitation facilities, total (P...  66.0518       NaN   

                                                Source  
0    World Health Organization (WHO) and United Nat...  
1    World Health Organization (WHO) and United Nat...  
2    World Health Organization (WHO) and United Nat...  
3    World Health Organization (WHO) and United Nat...  
4    World Health Organization (WHO) and United Nat...  
..                                                 ...  
257  World Health Organization (WHO) and United Nat...  
258  World Health Organization (WHO) and United Nat...  
259  World Health Organization (WHO) and United Nat...  
260  World Health Organization (WHO) and United Nat...  
261  World Health Organization (WHO) and United Nat...  

[262 rows x 7 columns]

In [10]:
df2.columns

Index(['Region/Country/Area', 'Unnamed: 1', 'Year', 'Series', 'Value',
       'Footnotes', 'Source'],
      dtype='object')

In [11]:
df [['Region/Country/Area', 'Unnamed: 1', 'Year', 'Series', 'Value']]

Region/Country/Area                     Unnamed: 1  Year  \
0                       1  Total, all countries or areas  2000   
1                       1  Total, all countries or areas  2005   
2                       1  Total, all countries or areas  2010   
3                       1  Total, all countries or areas  2015   
4                       1  Total, all countries or areas  2016   
...                   ...                            ...   ...   
1464                  716                       Zimbabwe  2005   
1465                  716                       Zimbabwe  2010   
1466                  716                       Zimbabwe  2015   
1467                  716                       Zimbabwe  2016   
1468                  716                       Zimbabwe  2017   

                                            Series    Value  
0     Percentage of individuals using the internet   6.5316  
1     Percentage of individuals using the internet  15.6777  
2     Percentage of individuals using the internet  28.7084  
3     Percentage of individuals using the internet  41.5491  
4     Percentage of individuals using the internet  44.5569  
...                                            ...      ...  
1464  Percentage of individuals using the internet   2.4000  
1465  Percentage of individuals using the internet   6.4000  
1466  Percentage of individuals using the internet  22.7428  
1467  Percentage of individuals using the internet  23.1200  
1468  Percentage of individuals using the internet  27.0555  

[1469 rows x 5 columns]

les 258 lignes portent sur des zones consolidées
A partir de la 259, par pays // Australia

Qu'est ce que je veux faire ?
Comparer dans le temps,  l'accès à l'eau potable et à internet, dans chaque aire géographique.
Année commune 2005, 2010, 2017